invoices.go

```diff
@@ -0,0 +1,121 @@
+package recurly
+
+import (
+	"encoding/xml"
+
+	"github.com/blacklightcms/go-recurly/types"
+)
+
+type (
+	// Invoice is an individual invoice for an account.
+	// The only fields annotated with XML tags are those for posting an invoice.
+	// Unmarshaling an invoice is handled by the custom UnmarshalXML function.
+	Invoice struct {
+		XMLName               xml.Name       `xml:"invoice,omitempty"`
+		AccountCode           string         `xml:"-"`
+		Address               Address        `xml:"-"`
+		SubscriptionUUID      string         `xml:"-"`
+		OriginalInvoiceNumber int            `xml:"-"`
+		UUID                  string         `xml:"-"`
+		State                 string         `xml:"-"`
+		InvoiceNumberPrefix   string         `xml:"-"`
+		InvoiceNumber         int            `xml:"-"`
+		PONumber              string         `xml:"po_number,omitempty"` // PostInvoice param
+		VATNumber             string         `xml:"-"`
+		SubtotalInCents       int            `xml:"-"`
+		TaxInCents            int            `xml:"-"`
+		TotalInCents          int            `xml:"-"`
+		Currency              string         `xml:"-"`
+		CreatedAt             types.NullTime `xml:"-"`
+		ClosedAt              types.NullTime `xml:"-"`
+		TaxType               string         `xml:"-"`
+		TaxRegion             string         `xml:"-"`
+		TaxRate               float64        `xml:"-"`
+		NetTerms              types.NullInt  `xml:"net_terms,omitempty"`                // PostInvoice param
+		CollectionMethod      string         `xml:"collection_method,omitempty"`        // PostInvoice param
+		TermsAndConditions    string         `xml:"terms_and_conditions,omitempty"`     // PostInvoice param
+		CustomerNotes         string         `xml:"customer_notes,omitempty"`           // PostInvoice param
+		VatReverseChargeNotes string         `xml:"vat_reverse_charge_notes,omitempty"` // PostInvoice param
+		LineItems             []Adjustment   `xml:"-"`
+		Transactions          []Transaction  `xml:"-"`
+	}
+)
+
+const (
```

Comment:
Fixed in [56f7a0](https://github.com/blacklightcms/go-recurly/pull/10/commits/70680f140c93eac4f28ac08599257fe42756f7a0)